tuxemon/core/map.py

```diff
@@ -253,7 +253,10 @@ def load(self, filename):
         self.size = self.data.width, self.data.height
 
         # Get the tile size of the map
-        self.tile_size = self.data.tilesets[0].tilewidth, self.data.tilesets[0].tileheight
+        if len(self.data.tilesets) > 0:
+            self.tile_size = self.data.tilesets[0].tilewidth, self.data.tilesets[0].tileheight
+        else:
+            self.tile_size = 16, 16
```

Comment:
use the value from PREPARE.  i think it is called `TILE_SIZE`

Reply:
Fixed.